spidev_module.c

```diff
@@ -173,12 +173,13 @@ PyDoc_STRVAR(SpiDev_write_doc,
 static PyObject *
 SpiDev_writebytes(SpiDevObject *self, PyObject *args)
 {
-	int		status;
-	uint16_t	ii, len;
-	uint8_t	buf[SPIDEV_MAXPATH];
-	PyObject	*obj;
-	PyObject	*seq;
-	char	wrmsg_text[4096];
+	int status;
+	uint16_t ii, len;
+	PyObject *obj;
+	PyObject *seq;
+	const uint32_t  spi_blk_maxsize = get_xfer_block_size();
+	uint8_t buf[spi_blk_maxsize];
+	char wrmsg_text[spi_blk_maxsize];
```

Comment:
Why does the message string buffer have the same size as the data buffer? This looks wrong.